homeassistant/components/jellyfin/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Jellyfin integration."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .config_flow import CannotConnect, InvalidAuth, create_client, validate_input
+from .const import DATA_CLIENT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Jellyfin from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    try:
+        client = create_client()
```

Comment:
Please move this line out of the try... except block too.

Reply:
Yes, of course. Done